sdk/test/NetStandard/UnitTests/Services/DynamoDBTests.cs

```diff
@@ -203,6 +203,7 @@ public async Task TestMockingTableClient_TransactWriteAsync()
         public void TestMockingTableClient_LoadTable_Throws()
         {
             var mockClient = new Mock<IAmazonDynamoDB>();
+            mockClient.Setup(x => x.Config).Returns(new AmazonDynamoDBConfig());
```

Comment:
This test was failing even before the merge (i.e. on the `HEAD` of `v4-development`). The issue was `Table.LoadTable` (which is tested below) now tries to apply our observability wrapper but the config object on our mock client was null (so we'd get a NPE instead of invalid operation).